ledger/block/src/verify.rs

```diff
@@ -27,6 +27,7 @@ use rayon::prelude::*;
 
 impl<N: Network> Block<N> {
     /// Ensures the block is correct.
+    #[cfg_attr(feature = "instrumentation", tracing::instrument(skip_all))]
```

Comment:
would it be possible to include the block's hash in the instrumentation, so that we could immediately tell which blocks potentially caused a slowdown?